Doc/howto/unicode.rst

```diff
@@ -578,7 +599,7 @@ Unicode filenames
 Most of the operating systems in common use today support filenames that contain
 arbitrary Unicode characters.  Usually this is implemented by converting the
 Unicode string into some encoding that varies depending on the system.  For
-example, Mac OS X uses UTF-8 while Windows uses a configurable encoding; on
+example, Mac OS X uses UTF-8.  Windows uses a configurable encoding; on
 Windows, Python uses the name "mbcs" to refer to whatever the currently
```

Comment:
In 3.6+ by default we use UTF-8 as the Windows file-system encoding. Internally UTF-8 is converted to native UTF-16LE and vice versa.

Reply:
Updated; thanks!